data/Localization/german_(germany)/global.ini

```diff
@@ -7901,7 +7901,7 @@ Frontend_Presence_Away=Abwesend
 Frontend_Presence_Busy=Beschäftigt
 Frontend_Presence_InMenu=Im Hauptmenü
 Frontend_Presence_Offline=Offline
-Frontend_Presence_OnSpectrum=Im Spektrum
+Frontend_Presence_OnSpectrum=In Spektrum
```

Comment:
Wäre nicht "Auf Spectrum" korrekter?

Reply:
Ich wollte erst In Spektrum online schreiben aber wenn man da Abwesend ist passts ja wieder nicht und "Auf Spektrum" habe ich auch gedacht konnte mich nicht entscheiden ^^

Reply:
Naja Spectrum ist eh falsch geschrieben xD

Reply:
Wait ich mach nochmal einen neuen Commit soll ich das doch besser "Auf Spectrum" nennen?

Reply:
Ich habe jetzt Spektrum wieder zu Spectrum gemacht aber ich finde "In" besser da man ja sagt "In Spectrum online"

Reply:
Ja lassen wir das jetzt erstmal so

Reply:
Passt